lib/csv_helper/loader.rb

```diff
@@ -99,6 +89,20 @@ def after_import_validations(records, failed_instances, options)
       end
     end
 
+    def run_after_import_batch_validations?(failed_instances)
+      # this a call to custom batch validation checks for large import CSVs
+      validator_klass = "#{klass.name}Validator".safe_constantize
+      run_validations = validator_klass.present? && defined? validator_klass.after_import_batch_validations
+
+      if run_validations
+        failed_instances.each do |record|
+          record.errors.add(:row, record.try(:csv_row))
```

Comment:
Should we change it from `record.try(:csv_row)` to `record.csv_row`?
Assuming more "batch validation" uploads/CSVs are going to be added, and they will all have a `csv_row` column on their associated tables, we want this to raise an exception if it doesn't exist (i.e. the column hasn't been added to the table).

Reply:
changed